Add warning on IdP deletion (#162)
mulesoft/docs-access-management: modules/ROOT/pages/map-users-roles-teams.adoc

== Map SSO Users to Teams

[NOTE]

> Deleting your IdP configuration will result in your associated team mappings to be deleted as well

Reviewer: ```suggestion
If you delete your IdP configuration, your associated team mappings are also deleted. 
```
